### PR TITLE
updating JSS plugin deps fixes #1467

### LINKED ISSUE #1467: jss-camel-case@6.1.0" has incorrect peer dependency "jss@^9.7.0" on React Styleguidist v10.2.0
Installing the latest version of react-styleguidist gives me:
```
warning "react-styleguidist > jss-camel-case@6.1.0" has incorrect peer dependency "jss@^9.7.0".
warning "react-styleguidist > jss-compose@5.0.0" has incorrect peer dependency "jss@^9.0.0".
warning "react-styleguidist > jss-default-unit@8.0.2" has incorrect peer dependency "jss@^9.4.0".
warning "react-styleguidist > jss-global@3.0.0" has incorrect peer dependency "jss@^9.0.0".
warning "react-styleguidist > jss-isolate@5.1.0" has incorrect peer dependency "jss@^9.0.0".
warning "react-styleguidist > jss-nested@6.0.1" has incorrect peer dependency "jss@^9.0.0".
```

Looks like these dependencies are outdated and have moved.
for e.g: [jss-camel-case](https://github.com/cssinjs/jss-camel-case) is now j[ss-plugin-camel-case](https://github.com/cssinjs/jss/tree/master/packages/jss-plugin-camel-case)

**Outcome?**
```
jss-camel-case@6.1.0
jss-compose@5.0.0
jss-default-unit@8.0.2
jss-global@3.0.0
jss-isolate@5.1.0
jss-nested@6.0.1
```
Need updating/changing

### PR DESCRIPTION
**Pull request by jasonwilliams**
@sapegin  hows this?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,260 +1,260 @@
...
-    "*.{ts,tsx,js,md,css}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "funding": {
-    "type": "opencollective",
-    "url": "https://opencollective.com/styleguidist"
-  }
+	"name": "react-styleguidist",
```
Comment (sapegin) [inline package.json]: Please revert indentation changes.

Comment (jasonwilliams) [reply package.json]: Done, i don't know why that happened as i use prettier and editorConfig, oh well

**Comment by elevatebart** [comment]
Hello @jasonwilliams,

Can you as well delete the typings in `src/typings/jss.d.ts`? 
They are now completely useless.

To do so, just delete the file ;-)

**Comment by sapegin** [comment]
Thanks, merging!

**Pull request merged**